lib/ts/recipe/emailpassword/components/features/resetPasswordUsingToken/index.tsx

```diff
@@ -90,14 +93,16 @@ class ResetPasswordUsingToken extends PureComponent<PropType, { token: string |
         };
 
         return (
-            <FeatureWrapper isEmbedded={this.getIsEmbedded()} useShadowDom={this.props.recipe.config.useShadowDom}>
-                <Fragment>
-                    {/* No custom theme, use default. */}
-                    {this.props.children === undefined && <ResetPasswordUsingTokenTheme {...props} />}
-                    {/* Otherwise, custom theme is provided, propagate props. */}
-                    {this.props.children && React.cloneElement(this.props.children, props)}
-                </Fragment>
-            </FeatureWrapper>
+            <ComponentOverrideContext.Provider value={componentOverrides}>
```

Comment:
- What if this feature is being used inside another feature? Then we will double add this provider.. is that an issue?
- Also, We need to only add this feature in case we are rendering `ResetPasswordUsingTokenTheme` component.. so it should be around that component and not this entire block?